conf/messages

```diff
@@ -941,7 +941,7 @@ case.v2.liability.reopen.button=Reopen case
 
 case.v2.liability.advanced_search.button=Search cases
 
-case.v2.correspondence.page_title=Correspondence case details
```

Comment:
can we keep those (misc and corrie and liability) the same as per Eszter's comment on the ticket please